resources/views/inputs/switch.blade.php

```diff
@@ -2,28 +2,32 @@
     x-data="{ value: {{ $default ?? 'false' }}, toggle() { this.value = !this.value; this.$refs['checkbox-livewire'].click(); }, focused: false }"
     class="flex items-center space-x-3"
 >
-    <span class="font-semibold {{ $labelClass ?? '' }}" :class="{ 'text-theme-secondary-500': !value }">
+    <span class="font-semibold {{ $labelClass ?? '' }}" :class="{ 'text-theme-secondary-500': value }">
```

Comment:
why did this change?

Reply:
No clues of how it worked before, but was just the wrong direction when I tried it, so changed.